fastlane/lib/fastlane/plugins/templates/action.rb.erb

```diff
@@ -0,0 +1,35 @@
+module Fastlane
+  module Actions
+    class <%= plugin_name.capitalize %>Action < Action
+      def self.run(params)
+        UI.message("The <%= plugin_name %> plugin is working!")
+      end
+
+      def self.description
+        %q{<%= summary %>}
```

Comment:
I don't mean to be obstructionist about this. I totally understand why you feel like the special syntax is confusing, so if you feel strongly that it should be changed, go for it! :smile: